components/brave_wallet_ui/components/desktop/views/nfts/components/nfts.tsx

```diff
@@ -257,6 +322,23 @@ export const Nfts = (props: Props) => {
         )
     }, [renderedList])
 
+  const renderGridViewItem = React.useCallback(
+    (nft: BraveWallet.BlockchainToken) => {
+      const assetId = getAssetIdKey(nft)
+
+      return (
+        <NFTGridViewItem
+          key={assetId}
+          token={nft}
+          onSelectAsset={() => onSelectAsset(nft)}
+          isTokenHidden={hiddenNftsIds.includes(assetId)}
+          isTokenSpam={allSpamNftsIds.includes(assetId)}
+        />
+      )
+    },
```

Comment:
If we inline this, its going to be repeated in `listUi`, `listUiByNetworks`, `listUiByAccounts`. That is the main reason I refactored it to a function to avoid the duplication.